src/quo2/components/icon_avatar.cljs

```diff
@@ -0,0 +1,78 @@
+(ns quo2.components.icon-avatar
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(def themes {:light {:primary   {:icon-color       colors/primary-50
```

Comment:
isn't this duplicated in different components?

Reply:
Yes, Do you think we can merge only one, and solve conflicts ?

Reply:
i guess it would be better to have only one place with common themes

Reply:
maybe we can create a directory in `quo2/components/` as `themes` (or some better name), and you can move all these similar styles as `quo2/components/themes/avatar-styles.cljs`. wdyt?

Reply:
or better, create `avatars` directory in `quo2/components` and move all avatars and `themes.cljs` inside that directory.

Reply:
Refactored